riscv/riscv.mk.in

```diff
@@ -1395,6 +1395,10 @@ riscv_insn_ext_zvksh = \
 	vsm3c_vi \
 	vsm3me_vv \
 
+riscv_insn_ext_zimop = \
+    mop_r_N \
```

Comment:
This file uses tabs--probably for bad reasons--but please be consistent with the surrounding spacing.

Reply:
Fixed it. My editor was setup to replace tabs with space.